### PR TITLE
feat: use amcharts5 in cost analysis

### PR DESCRIPTION
**Pull request by yuda110**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
amcharts4 -> amcharts5

스크린샷 2023-08-25 오후 5 39 08

### CODE REVIEW

File: apps/web/src/services/cost-explorer/cost-analysis/modules/CostAnalysisStackedColumnChart.vue [hunk 1]
```diff
@@ -42,225 +28,153 @@ import {
...
+    currency: CURRENCY.USD,
+    currencyRates: () => ({}) as CurrencyRates,
+    printMode: false,
+});
+const emit = defineEmits<{(e: 'update:chart', value): void;
+    (e:'rendered'): void;
+}>();
+
+const chartContext = ref<HTMLElement | null>(null);
+const chartHelper = useAmcharts5(chartContext);
```
Comment (sulmoJ) [inline apps/web/src/services/cost-explorer/cost-analysis/modules/CostAnalysisStackedColumnChart.vue:70]: praise: Crazy Helper

**Review by sulmoJ — APPROVED** [review]
Using AmChart 5 significantly reduced the amount of code and made it easier to review!
LGTM!

File: apps/web/src/services/cost-explorer/cost-analysis/modules/CostAnalysisStackedColumnChart.vue [hunk 2]
```diff
@@ -1,39 +1,25 @@
...
-    reactive, toRefs, watch,
+    nextTick,
+    reactive, ref, watch,
 } from 'vue';
-import type { Vue } from 'vue/types/vue';
 
-import type { XYChart } from '@amcharts/amcharts4/charts';
-import * as am4charts from '@amcharts/amcharts4/charts';
-import * as am4core from '@amcharts/amcharts4/core';
+import type { TimeUnit } from '@amcharts/amcharts4/core';
```
Comment (piggggggggy) [inline apps/web/src/services/cost-explorer/cost-analysis/modules/CostAnalysisStackedColumnChart.vue:7]: question: will this be replaced with v5 after?

**Review by piggggggggy — APPROVED** [review]
LGTM!

File: apps/web/src/services/cost-explorer/cost-analysis/modules/CostAnalysisStackedColumnChart.vue (hunk 2 above)
Comment (yuda110) [reply apps/web/src/services/cost-explorer/cost-analysis/modules/CostAnalysisStackedColumnChart.vue:7]: wow thanks😨

**Pull request merged**